js/animation.js

```diff
@@ -0,0 +1,321 @@
+            this.backButton2.style.color = clrFont2;
+            this.backButton3.style.color = clrFont3;
+            return true;
+        }
+
+        changeData(serv) {
+        if ([`Head${serv}`] in aboutService) {
+            this.serviceHead.innerHTML = aboutService[`Head${serv}`];
+            this.serviceDescr.innerHTML = aboutService[`Descr${serv}`];
+            return true;
+        }
+        return false;
+        }
+
+        servIconVisible(vis1, vis2, vis3) {
+            this.servIcon1.style.display = vis1;
+            this.servIcon2.style.display = vis2;
+            this.servIcon3.style.display = vis3;
+            return true;
+        }
+
+        changeDescrService = (data, but, visible) => {
+            this.changeData(data);
+            this.backBut(...but);
+            this.servIconVisible(...visible);
+        }
+
+        bodyValue = {
+            'but': ['.s1', '.s2', '.s3'],
+            'valArr1': ['Serv1', ['black', '', '', 'white', 'black', 'black'], ['block', 'none', 'none']],
+            'valArr2': ['Serv2', ['', 'black', '', 'black', 'white', 'black'], ['none', 'block', 'none']],
+            'valArr3': ['Serv3', ['', '', 'black', 'black', 'black', 'white'], ['none', 'none', 'block']],
+        }
+    }
+
+    const desrcServ = new ServiceDesrc;
+
+    desrcServ.bodyValue.but.map(function (elem) {
+        document.querySelector(elem).addEventListener('click', function () {
+            desrcServ.changeDescrService(...desrcServ.bodyValue[`valArr${desrcServ.bodyValue.but.indexOf(elem)+1}`]);
+        })
+    })
+}
+
+//scroll icon-picture
+{
+    class ScrollIconPicture {
+        constructor() {
+            this.numHead = 1;
+            this.numText = 1;
+            this.headDescrPict = document.querySelectorAll('.head-pd-descr')
+            this.DescrPict = document.querySelectorAll('.text')
+            this.updownElem1 = document.querySelector('.pct1');
+            this.updownElem2 = document.querySelector('.pct2');
+            this.updownElem3 = document.querySelector('.pct3');
+            this.hiddenElement1 = document.querySelector(".phoneDescr");
+            this.hiddenElement2 = document.querySelector(".laptopDescr");
+            this.hiddenElement3 = document.querySelector(".watchDescr");
+
+            this.hidElem = [this.hiddenElement1, this.hiddenElement2, this.hiddenElement3];
+            this.upDownElem = [this.updownElem1, this.updownElem2, this.updownElem3];
+        }
+
+        handleButtonClick(elem) {
+            elem.scrollIntoView({ block: "center", behavior: "smooth" });
+            return true;
+        }
+
+    }
+
+    const scrollWin = new ScrollIconPicture;
+
+    scrollWin.headDescrPict.forEach((elem) => {
+        elem.innerHTML = descrProdPict[`head${scrollWin.numHead++}`];
+    })
+
+    scrollWin.DescrPict.forEach((elem) => {
+        elem.innerHTML = descrProdPict[`descr${scrollWin.numText++}`];
+    })
+
+    scrollWin.upDownElem.map(function(elem) {
```

Comment:
Why you don't assign result of .map call abywhere? map - maps one array to another by it's semantics